src/main/java/org/sunbird/workflow/postgres/repo/WfStatusRepo.java

```diff
@@ -103,5 +103,8 @@ public interface WfStatusRepo extends JpaRepository<WfStatusEntity, String> {
 
     @Query(value = "select update_field_values, wf_id, comment, lastupdated_on, current_status from wingspan.wf_status where service_name = ?1 and userid = ?2 order by lastupdated_on desc" , nativeQuery = true)
     List<Object[]> findWfFieldsForUserV2(String servicename , String userId);
+
+    @Query(value = "select * from wingspan.wf_status where service_name = ?1 and current_status = ?2 and dept_name = ?3", nativeQuery = true)
+    List<WfStatusEntity> getListOfDistinctApplicationUsingDept(String serviceName, String currentStatus, String deptName);
```

Comment:
the query is not actually returning distinct records -- do update the name of the method